scripts/incguard-checker/check-include-guard.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
```

Comment:
Thanks for doing this work, but I was wanting more of an actual build onboarded that was breaking without these changes, than a separate tool/script. Additionally, all scripts in our code base use PowerShell instead of Python.